test/commands/add/activity_spec.rb

```diff
@@ -29,12 +29,31 @@ def date_parsing_specs(test_stdout: true)
       it { stdout_only "Activity added: \"2017-03-02: #{description}\"" } if test_stdout
     end
 
-    describe "when date is natural language" do
+    describe "when date is natural language and in full" do
       let(:date) { "February 23rd, 2017" }
 
       it { line_added "- 2017-02-23: #{description}" }
       it { stdout_only "Activity added: \"2017-02-23: #{description}\"" } if test_stdout
     end
+
+    describe "when date is natural language and only month and day" do
+      mocked_time = Time.new(2017, 8, 1) # First of August
```

Comment:
Would you mind changing the mocked year to 2016 or earlier? If the mocking doesn't work the way we think it does (or, for instance, Chronic's internals change and it stops using `Time.now`), I'd like to make sure that this test fails rather than passing because it happens to be 2017 now.

Does that make sense?